### PR TITLE
Feature/add console colorize for pretest methods

### PR DESCRIPTION
**Pull request by askonev**

### CODE REVIEW

**Review by ShockwaveNN — CHANGES_REQUESTED** [review]
I think rubocop is not happy

File: helpers/file_helper.rb [hunk 1]
```diff
@@ -26,10 +26,10 @@ def self.file_rename(file_name)
   # @return [nil]
   def self.check_temp_dir(dir_name)
     if File.exist? dir_name
-      puts "Directory #{dir_name} exist?: true"
+      OnlyofficeLoggerHelper.green_log "Directory #{dir_name} exist?: true"
```
Comment (ShockwaveNN) [inline helpers/file_helper.rb:29]: I think this log maybe use default color (use method `log`), since it's not very critical information

**Review by ShockwaveNN — CHANGES_REQUESTED** [review]

File: helpers/file_helper.rb (hunk 1 above)
Comment (askonev) [reply helpers/file_helper.rb:29]: Yes, I agree. Check for folder existence added

**Comment by askonev** [comment]
> I think rubocop is not happy

I can

**Comment by askonev** [comment]
"miss clicks" invented by the devil

**Review by ShockwaveNN — APPROVED** [review]

**Pull request merged**